Sources/Sentry/SentrySystemEventBreadcrumbs.m

```diff
@@ -177,6 +181,58 @@ - (void)initScreenshotObserver
                                                  name:UIApplicationUserDidTakeScreenshotNotification
                                                object:nil];
 }
+
+- (NSNumber *_Nullable)storedTimezoneOffset
+{
+    SentryAppStateManager *appStateManager =
+        [SentryDependencyContainer sharedInstance].appStateManager;
+    SentryAppState *currentState = [appStateManager loadCurrentAppState];
+    return currentState.timezoneOffset;
+}
+
+- (void)initTimezoneObserver
+{
+    // Detect if the stored timezone is different from the current one;
+    // if so, then we also send a breadcrumb
+    NSNumber *_Nullable storedTimezoneOffset = [self storedTimezoneOffset];
+
+    if (storedTimezoneOffset == nil) {
+        [self updateStoredTimezone];
+    } else if (storedTimezoneOffset.doubleValue != [NSTimeZone localTimeZone].secondsFromGMT) {
+        [self timezoneEventTriggered];
+    }
+
+    // Posted when the timezone of the device changed
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(timezoneEventTriggered)
+                                                 name:NSSystemTimeZoneDidChangeNotification
+                                               object:nil];
+}
+
+- (void)timezoneEventTriggered
+{
+    SentryBreadcrumb *crumb = [[SentryBreadcrumb alloc] initWithLevel:kSentryLevelInfo
+                                                             category:@"device.event"];
+    crumb.type = @"system";
+    crumb.data = @{
+        @"action" : @"TIMEZONE_CHANGE",
+        @"previous" : [self storedTimezoneOffset] ?: @(0),
+        @"new" : @([NSTimeZone localTimeZone].secondsFromGMT)
```

Comment:
`h`: I think we have to clarify that this is `secondsFromGMT`? @brustolin what do you think?
```suggestion
        @"previous_seconds_from_gmt" : [self storedTimezoneOffset] ?: @(0),
        @"current_seconds_from_gmt" : @([NSTimeZone localTimeZone].secondsFromGMT)
```